liferay/liferay-ckeditor
# fix: add skin files for plugins that support skinning

In [85bfdebb7f23872a9d6dfb94d4de49325d6df618](https://github.com/liferay/liferay-ckeditor/pull/67/commits/85bfdebb7f23872a9d6dfb94d4de49325d6df618),
we created a new (`moono-lexicon`) skin for CKEditor,
but some plugins also support skinning and can't be used correctly
with our custon `moono-lexicon` skin.

These are the plugins that also need css files in a `moono-lexicon`
directory in order to work correctly

| Plugin name  | Skin directory |
| ------------ | -------------- |
| a11ychecker  | ./ckeditor-dev/plugins/a11ychecker/skins |
| balloonpanel | ./ckeditor-dev/plugins/balloonpanel/skins |
| balloontoolbar | ./ckeditor-dev/plugins/balloontoolbar/skins |
| scayt | ./ckeditor-dev/plugins/scayt/skins |
| wsc | ./ckeditor-dev/plugins/wsc/skins |

At the moment, we're using the files that come with the default
`moono-lisa` skin, and if we need to we can customize these to our liking

Test plan:
- With the current version of `liferay-ckeditor` in `liferay-portal`,
  try to use the `balloonpanel` plugin and see it fail because of missing css files

  The error looks like this
  ```
  Refused to apply style from
  'http://localhost:8080/o/frontend-editor-ckeditor-web/ckeditor/plugins/balloonpanel/skins/moono-lexicon/balloonpanel.css'
  because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
  ```

- Run `./ck.sh build`
- Try to use the `balloonpanel` plugin and see it working correctly


Closes https://issues.liferay.com/browse/IFI-1858
